labelbox/schema/model_run.py

```diff
@@ -446,6 +449,66 @@ def export_labels(
                          self.uid)
             time.sleep(sleep_time)
 
+    """
+    Creates a model run export task with the given filter and returns the task.
+    
+    >>>    export_task = export_labels_v2("my_export_task", filter={"media_attributes": True})
+    
+    """
+
+    def export_labels_v2(self, task_name: str,
+                         filter: Optional[ModelRunExportFilter]) -> Task:
+        mutation_name = "exportDataRows"
+        create_task_query_str = """mutation exportDataRowsPyApi($input: ExportDataRowsInput!){
+          %s(input: $input) {taskId} }
+          """ % (mutation_name)
+        params = {
+            "input": {
+                "taskName": task_name,
+                "filters": {
+                    "modelRunIds": [self.uid],
+                    "projectIds": []
+                },
+                "params": {
+                    "includeAttachments":
+                        filter["attachments"]
```

Comment:
It is also weird that the gql fields are different than the name we expose in the SDK. I can see a future where this causes confusion :). 

We should make these the same.

Reply:
Would you update SDK or the API? Also, should SDK use camelCase or snake_case?

Reply:
We should update the api. I think it is fine if the api casing is camelcase and sdk is snake case just that the names should be the same.